### ISSUE TITLE
CoreData: warning: Unable to load class 

### ISSUE THREAD

**Issue by GoogleCodeExporter** [gzog/analytics-issues#486]
```
---------------------------------------------------------------------------
NOTE: This issue tracking system is for Google Analytics developer products
only.
If you are not a developer/programmer visit:
http://www.google.com/analytics/support.html
---------------------------------------------------------------------------
Name of affected component: Mobile Tracking

Name of related library and version, if applicable (e.g. Android, iOS,
Snippets, etc.): Google Analytics Services SDK for iOS 3.09

Issue summary:
When starting the app in emulator, see this in the console:

CoreData: warning: Unable to load class named 'GAIProperty' for entity 
'GAIProperty'.  Class not found, using default NSManagedObject instead.
CoreData: warning: Unable to load class named 'GAIHit' for entity 'GAIHit'.  
Class not found, using default NSManagedObject instead.


Steps to reproduce issue:
1. Use Google Analytics on Xcode 6 with iOS SDK 8 targeting iOS 7.  Run the app.
2.
3.

Expected output:
No errors in console

Actual results:
Errors as listed above

Notes:
If these warnings are harmless, would rather not see the warnings.  If these 
warnings mean something, document how we should migrate our projects.
```

Original issue reported on code.google.com by `ellucian...@gmail.com` on 4 Sep 2014 at 5:49

**Comment by GoogleCodeExporter** [comment]
```
I have the same problem. 
```

Original comment by `francesc...@gmail.com` on 16 Sep 2014 at 11:36

**Comment by GoogleCodeExporter** [comment]
```
I experience the same issue, was going to submit app to apple for review today, 
they may reject based on these errors.
```

Original comment by `Jason.Br...@googlemail.com` on 19 Sep 2014 at 6:16

**Comment by GoogleCodeExporter** [comment]
```
I get this error every time I do a build.  I have submitted my app 2 times 
(first time for original submission and second time for an update) and haven't 
been rejected for this yet.
```

Original comment by `fba...@gmail.com` on 19 Sep 2014 at 6:54

**Comment by GoogleCodeExporter** [comment]
```
I do get the same error only when I try to run app in ios 8 device and 
simulator. The app fails to run on ios 8 as my application needs to get 
information from coredata. The location of *sqlite seems to have changed with 
ios 8. I do not use any custom location. There are no issues with io7.1 app.
```

Original comment by `ramasesh...@gmail.com` on 22 Sep 2014 at 3:34

**Comment by GoogleCodeExporter** [comment]
```
same issue with me... Xcode 6 and simulator running ios8
```

Original comment by `bbr-supp...@limitless.co.uk` on 22 Sep 2014 at 11:12

**Comment by GoogleCodeExporter** [comment]
```
Did someone submitted the app with this warnings and got approved?
```

Original comment by `jcesarmo...@gmail.com` on 6 Oct 2014 at 1:32

**Comment by GoogleCodeExporter** [comment]
```
I submitted an app with this error and it was approved.
```

Original comment by `dantrous...@gmail.com` on 6 Oct 2014 at 1:39

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
This message can be safely ignored.  It'll be fixed in the next version of the 
SDK.
```

Original comment by `pfrise...@google.com` on 21 Oct 2014 at 6:36
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
This has been fixed in v3.10.
```

Original comment by `pfrise...@google.com` on 14 Nov 2014 at 7:47
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
I'm still receiving this error after upgrading to v3.10:
CoreData: warning: Unable to load class named 'GAIProperty' for entity 
'GAIProperty'.  Class not found, using default NSManagedObject instead.

CoreData: warning: Unable to load class named 'GAIHit' for entity 'GAIHit'.  
Class not found, using default NSManagedObject instead.
```

Original comment by `fba...@gmail.com` on 26 Nov 2014 at 6:36

**Comment by GoogleCodeExporter** [comment]
```
Updating today to Yosemite + XCode 6 + Google Analytics 3.10.

Same errors received (i cleaned my project after the update).

This bug is not fixed for me.
```

Original comment by `gdemu...@gmail.com` on 26 Jan 2015 at 7:35

**Comment by GoogleCodeExporter** [comment]
```
This issue is closed but a new issue was opened for v3.10.
https://code.google.com/p/analytics-issues/issues/detail?id=539

```

Original comment by `fba...@gmail.com` on 26 Jan 2015 at 3:53
